cg/apps/scout/scout_export.py

```diff
@@ -1,32 +1,41 @@
 """Schemas for scout serialisation"""
 
 from datetime import datetime
-from typing import List, Optional
+from typing import List, Optional, Dict
 
 from pydantic import BaseModel, Field, validator
 from typing_extensions import Literal
 
+from cg.constants.gene_panel import GENOME_BUILD_37
+from cg.constants.subject import (
+    PlinkGender,
+    Gender,
+    PlinkPhenotypeStatus,
+    RelationshipStatus,
+)
+from cg.constants.pedigree import Pedigree
+
 
 class Individual(BaseModel):
     bam_file: Optional[str] = None
     individual_id: str
-    sex: Literal["0", "1", "2", "other"]
+    sex: Literal[PlinkGender.UNKNOWN, PlinkGender.MALE, PlinkGender.FEMALE, Gender.OTHER]
     father: Optional[str]
     mother: Optional[str]
-    phenotype: Literal[1, 2, 0]
+    phenotype: PlinkPhenotypeStatus
     analysis_type: str = "wgs"
 
-    @validator("father", "mother")
-    def convert_to_zero(cls, v):
-        if v is None:
-            return "0"
-        return v
+    @validator(Pedigree.FATHER, Pedigree.MOTHER)
+    def convert_to_zero(cls, value):
+        if value is None:
+            return RelationshipStatus.HAS_NO_PARENT
+        return value
 
-    @validator("sex")
-    def convert_sex_to_zero(cls, v):
-        if v == "other":
-            return "0"
-        return v
+    @validator(Pedigree.SEX)
+    def convert_sex_to_zero(cls, value):
+        if value == Gender.OTHER:
+            return PlinkGender.UNKNOWN
```

Comment:
I am not familiar with Pink, so this might be correct, but it sounds counter intuitive that if gender is "other" then return that it is "unknown". Double check that this is intended.

Reply:
Yes, It is rather confusinf but it has to do with me not wanting to go out of scope and make to many unrelated changes and there are 2 different syntaxex of gender specification in cg/scout right now. PLINKs using ints and a  more readable using strs. I think we should clean this up in a later PR